samples/hybrid-connections/node/hyco-websocket-tunnel/lib/tunnel.js

```diff
@@ -78,7 +78,8 @@ exports.createTunnel = function(wsServerAddr, token, credentials, listen, forwar
       tcpSock.destroy();
     });
 
-    var url = wsServerAddr; //+ '&port=' + forward.port + '&host=' + forward.host;
+    //var url = wsServerAddr; //+ '&port=' + forward.port + '&host=' + forward.host;
+    var url = wsServerAddr + '&port=' + forward.port + '&host=' + forward.host;
```

Comment:
>'&port=' + forward.port + '&host=' + forward.host [](start = 29, length = 49)

where is the code which interprets the port and host name from the query string?